# Generated by Django 4.0 on 2021-12-30 17:59

from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('user', '0001_initial'),
        ('campaign', '0001_initial'),
    ]

    operations = [
        migrations.CreateModel(
            name='CampaignRanking',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('win', models.IntegerField(db_column='win', default=0, error_messages={'blank': '...', 'invalid_choice': '...', 'max_lenght': '...', 'null': '...', 'required': '...', 'unique': '...'}, verbose_name='win')),
                ('lose', models.IntegerField(db_column='lose', default=0, error_messages={'blank': '...', 'invalid_choice': '...', 'max_lenght': '...', 'null': '...', 'required': '...', 'unique': '...'}, verbose_name='lose')),
                ('create_at', models.DateTimeField(auto_now_add=True, db_column='create_at', error_messages={'blank': '...', 'invalid_choice': '...', 'max_lenght': '...', 'null': '...', 'required': '...', 'unique': '...'}, verbose_name='Created At')),
                ('update_at', models.DateTimeField(auto_now=True, db_column='update_at', error_messages={'blank': '...', 'invalid_choice': '...', 'max_lenght': '...', 'null': '...', 'required': '...', 'unique': '...'}, verbose_name='Update At')),
                ('user', models.OneToOneField(db_column='user', error_messages={'blank': '...', 'invalid_choice': '...', 'max_lenght': '...', 'null': '...', 'required': '...', 'unique': '...'}, on_delete=django.db.models.deletion.CASCADE, to='user.user', verbose_name='User')),
            ],
            options={
                'verbose_name_plural': 'Campaign Ranking',
                'db_table': 'CampaignRanking',
            },
        ),
        migrations.DeleteModel(
            name='BestUsersInMission',
        ),
    ]
